react/features/base/participants/reducer.js

```diff
@@ -241,6 +245,13 @@ ReducerRegistry.register('features/base/participants', (state = DEFAULT_STATE, a
             };
         }
 
+        if (isLocalScreenShare) {
+            return {
+                ...state,
+                localScreenShare: participant
```

Comment:
does it need to be the entire participant?

Reply:
I don't believe so, just an id should be enough for how we are using it

Reply:
this is really a local participant and there are other means for getting the local participant (there's a selector for that)

Reply:
I'll revisit just storing just the participantId instead of the entire participant in a follow up PR